plugins/kotlin-dataframe/testData/box/wrongReceiver.kt

```diff
@@ -0,0 +1,14 @@
+import org.jetbrains.kotlinx.dataframe.annotations.DataSchema
+import org.jetbrains.kotlinx.dataframe.api.*
+
+
+@DataSchema
+data class Record(val a: String, val b: Int)
+
+fun box(): String {
+    val df = List(10) { Record(it.toString(), it) }.let { dataFrameOf(*it.toTypedArray()) }
+    val aggregate = df.pivot { b }.aggregate {
+        this.add("c") { 123 }.c
```

Comment:
which part failed before and is now solved?

Reply:
execution of this expression `this.add("c") { 123 }.c` failed due to wrong receiver in generated bytecode. `c` extension property used `this@aggregate` instead of scope it comes from